catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -150,9 +157,38 @@ public void init() {
   private CqlQueryResponse executeCqlQuery(CqlRequest cqlRequest)
       throws UnsupportedQueryException, SourceUnavailableException, FederationException {
     QueryRequest request = cqlRequest.createQueryRequest(catalogFramework.getId(), filterBuilder);
-
     Stopwatch stopwatch = Stopwatch.createStarted();
-    QueryResponse response = catalogFramework.query(request);
+
+    List<QueryResponse> responses = Collections.synchronizedList(new ArrayList<>());
```

Comment:
❓ Do you really want a synch list here and not something slightly more performant, like a `ConcurrentQueue`, maybe?

Reply:
SynchronizedList should be fine here as the overall number of additions to the list should be relatively small. We can always address in the future if this becomes a point of particular slowdown.